### PR TITLE
local setup link changed

### PR DESCRIPTION
**Pull request by Paarmita**
In your PR did you:

  - [ ] Include a description of the changes?
  - [ ] Mention the issue the PR addresses?
  - [ ] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [ ] Add and/or update specs for your code?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ All developers are welcome to contribute to the codebase.  We ask that, if possi
 * Sendgrid (for outbound email).
 * SSL using [Let's Encrypt](letsencrypt.org) and [Substrakt](https://github.com/substrakt/letsencrypt-heroku) at [End Bias Certificate Manager](https://endbias-certificate-manager.herokuapp.com/).
 
-For setting up a local development environment [here](docs/README.md)
+For setting up a local development environment [here](https://github.com/EBWiki/EBWiki/blob/docs/docs/SETUP_LOCALLY.md)
```
Comment (rlgreen91) [inline README.md]: Does this not work with a relative link?  It should work if you link to `docs/SETUP_LOCALLY.md`

Comment (Paarmita) [reply README.md]: Yeah, it will work with a relative link also

Comment (rlgreen91) [reply README.md]: Paarmita,  what's the status on this?  Is there anything blocking you from making this change?

Comment (Paarmita) [reply README.md]: @rlgreen91 this link also work.So I think it's done.

**Comment by Paarmita** [comment]
@rlgreen91 corrected the link.

**Pull request merged**